browser/js/dashboard/newDashboard/newDash.controller.js

```diff
@@ -0,0 +1,15 @@
+app.controller('newDashboardCtrl', function (userFactory,$state,$uibModalInstance, $scope) {
+	
+	// $scope.form = {
+	// 	name: dashbaord.name, 
+	// 	description: dashboard.description
+	// }
```

Comment:
I've always been curious about that, and about the same comment for console.log.  If the code is a work in progress, why not leave in tracing and notes-to-self about what to do next.  Is the argument that unfinished work shouldn't be merged to the mainline?  Or pushed at all?